test/game_connection/connector/test_echoes_remote_connector.py

```diff
@@ -234,13 +233,13 @@ async def test_find_missing_remote_pickups_give_pickup(connector: EchoesRemoteCo
 
 
 @pytest.mark.asyncio
-async def test_patches_for_pickup(connector: EchoesRemoteConnector, version: EchoesDolVersion, mocker):
+async def test_patches_for_pickup(connector: EchoesRemoteConnector, version: EchoesDolVersion, mocker, echoes_item_database):
     # Setup
     mock_item_patch: MagicMock = mocker.patch(
         "randovania.games.prime.all_prime_dol_patches.adjust_item_amount_and_capacity_patch")
 
     db = connector.game.resource_database
-    pickup = PickupEntry("Pickup", 0, ItemCategory.MISSILE, ItemCategory.MISSILE, progression=tuple(),
+    pickup = PickupEntry("Pickup", 0, echoes_item_database.item_categories["missile"], echoes_item_database.item_categories["missile"], progression=tuple(),
```

Comment:
A bunch of these tests don't care about the category. Could use a new fixture of "generic_item_category".